docs/tutorials/AddRemark.md

```diff
@@ -343,25 +343,25 @@ save it with `Model#setPerson()`.
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = new Person(
-                personToEdit.getName(), personToEdit.getPhone(), personToEdit.getEmail(),
-                personToEdit.getAddress(), remark, personToEdit.getTags());
+        Person studentToEdit = lastShownList.get(index.getZeroBased());
+        Person editedStudent = new Person(
```

Comment:
Failed to rename `Person` class here. I'm afraid other parts of the code might also have failed to rename.